src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -242,7 +242,29 @@ private static string GetDisplayText(ResourceViewModel resource)
         {
             stateText = $" ({resource.State})";
         }
-        return $"{resource.Name}{stateText}";
+        var resourceName = resource.DisplayName;
```

Comment:
Console log and telemetry pages should display the same text in the dropdown. I think the console log dropdown should use the same format as telemetry.

Reply:
If it means, use the OTel "service id" throughout the dashboard UI, I disagree. It is not meaningful to the user. The process ID works much better in this respect.